pkg/pdp/smartcontracts/verifier.go

```diff
@@ -55,6 +56,10 @@ func NewVerifierContract(backend bind.ContractBackend) (Verifier, error) {
 	}, nil
 }
 
+func (v *verifierContract) MaxPieceSizeLog2(ctx context.Context) (*big.Int, error) {
+	return v.verifier.MAXPIECESIZELOG2(&bind.CallOpts{Context: ctx})
```

Comment:
I don't know what this does, but it looks like it could be an expensive call? If that's the case, the value of `MaxPiecesSizeLog2` could be cached/memoized to avoid doing it every time. I'm assuming it's not a value that will change that often.

Reply:
https://github.com/storacha/piri/issues/308